LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Openverse
+Copyright (c) Automattic
```

Comment:
```suggestion
Copyright (c) Openverse
```

I've gotten final confirmation on this text, @dhruvkb. We can re-open this issue.